website/content/docs/enterprise/sentinel/index.mdx

```diff
@@ -42,22 +48,89 @@ Sentinel execution should be considered to be significantly slower than normal
+| -------------- | -------------------------------------------------------------------------- |
+| advisory       | The policy is allowed to fail. Can be used as a tool to educate new users. |
+| soft-mandatory | The policy must pass unless an override is specified.                      |
+| hard-mandatory | The policy must pass.                                       |
+
+
 ## Policy evaluation
 
-During evaluation, all policy types, if they exist, must grant access.
-Evaluation uses the following logic:
+Vault evaluates incoming requests against policies of all types that are
+applicable.
 
-1. If the request is unauthenticated, skip to step 3. Otherwise, evaluate the
-   token's ACL policies. These must grant access; as always, a failure to be
-   granted capabilities on a path via ACL policies denies the request.
-2. RGPs attached to the token are evaluated. All policies must pass according
+![Policy evaluation](/img/diagram-policy-evaluation-workflow_light.png#light-theme-only)
+![Policy evaluation](/img/diagram-policy-evaluation-workflow_dark.png#dark-theme-only)
+
+1. If the request is unauthenticated, skip to evaluating the EGPs. Otherwise,
+   evaluate the token's ACL policies. These must grant access; as always, a
+   failure to be granted capabilities on a path via ACL policies denies the
+   request.
+2. Evaluate RGPs attached to the cliet token. All policies must pass according
    to their enforcement level.
-3. EGPs set on the requested path, and any prefix-matching EGPs set on
-   less-specific paths, are evaluated. All policies must pass according to
-   their enforcement level.
+3. Evaluate EGPs set on the requested path, and any prefix-matching EGPs set on
+   less-specific paths, are evaluated. All policies must pass according to their
+   enforcement level.
 
 Any failure at any of these steps results in a denied request.
 
+### RGPs and namespaces
+
+Policies, auth methods, secrets engines, and tokens are locked into the
+[namespace](/vault/docs/enterprise/namespaces) they are created in. However,
+identity groups can pull in entities and groups from other namespaces.
+
+<Tip>
+
+Refer to the [Set up entities and groups section of the Secure Multi-Tenancy
+with Namespaces
+tutorial](/vault/tutorials/enterprise/namespaces#set-up-entities-and-groups) for
+a step-by-step instruction.
+
+</Tip>
+
+Consider the following scenario. 
+
+The training namespace is a child namespace of the education namespace. The "Sun
+Shine" entity created in the education namespace is a member of the "Tester"
+group which is defined in the training namespace. The group members inherit the
+group-level policy.
+
+![Relationship](/img/diagram-rgp-namespace_light.png#light-theme-only)
+![Relationship](/img/diagram-rgp-namespace_dark.png#dark-theme-only)
+
+While ACL policies and EGPs set rules on a specific path, an RGP may not have a
+target path. RGPs are tied to tokens, identity entities, or identity groups that
```

Comment:
The phrasing "may not" is open to multiple interpretations - to a reader without existing subject matter expertise, they may be uncertain whether the "it is uncertain whether it does or not" meaning or "it is not permitted" meaning is intended. Using "does not" would avoid the potential ambiguity.